.circleci/config.yml

```diff
@@ -1,7 +1,3 @@
-node4Environment: &node4Environment
```

Comment:
babel-plugin-module-resolver@3 does not support node@4. Either we stay on the old version of remove resting on node@4. Note: this does not remove the support of the library for node@4.